### PR TITLE
added directory to mount volume to

### PR DESCRIPTION
**Pull request by Domgartner**

### CODE REVIEW

**Comment by abarcomb-work** [comment]
Pull requests should contain useful information about the reasoning behind the changes for the repository maintainer to enable them to prioritize PRs. This is part of the collaborative workflow.

**Comment by abarcomb-work** [comment]
The PR should have been made to the ensf400-ws24 branch, not to main.

File: .devcontainer/docker-compose.yml [hunk 1]
```diff
@@ -11,10 +11,8 @@ services:
     #   context: .
     #   dockerfile: .devcontainer/Dockerfile
 
-    volumes:
-      # Update this to wherever you want VS Code to mount the folder of your project
-      - ..:/workspaces:cached
-      - /var/run/docker.sock:/var/run/docker.sock
+# Mount the volume to this directory
+    volumes: /lab04volume
```
Comment (growlf) [inline .devcontainer/docker-compose.yml:15]: this is an invalid mount directive and will not build or run correctly.

File: docker-compose2.yml
Comment (growlf) [inline docker-compose2.yml:1]: tthis seems o be comprised of code and changes that already exist in other branches, and so does not contribute anything new.

File: tests/baseline_test.py
Comment (growlf) [inline tests/baseline_test.py:1]: seems to be comprised of code and changes that already exist in other branches, and so does not contribute anything new.

**Pull request closed without merging**